### DISCUSSION TITLE
Switch to from ytdl-core-discord

### DISCUSSION THREAD

**Discussion by dylanjamesdev** [eritislami/evobot#279] [Ideas]
I'm assuming that `ytdl-core-discord` is deprecated since the repo is no longer a thing, I'm having a bunch of errors linked in other issues and topics in regards to that node module, would it be possible to switch to the regular ytdl-core or could you instruct me how to do it effectively? 

Are you able to replicate the same issue when playing music videos?
<img width="246" alt="Screen Shot 2020-10-27 at 9 23 40 AM" src="https://user-images.githubusercontent.com/55116398/97308895-c1e1ca00-1837-11eb-8210-2234da9f2521.png">

--Edit--
https://github.com/github/dmca/blob/master/2020/10/2020-10-23-RIAA.md
I came across this, I believe they went to youtube because the player works on lyric videos but not official (copyrighted) videos. Curious to see if there's another alternative we would be able to use

**Comment by paraswtf** [comment]
![image](https://user-images.githubusercontent.com/63962807/97338066-09536f00-18a7-11eb-9cb7-8c53d8a6e906.png)
But it was updated 4 days ago 🤔

**Reply by dylanjamesdev** [reply]
It was also under a DMCA takedown 3 days ago from today.

**Reply by eritislami** [reply]
> It was also under a DMCA takedown 3 days ago from today.

ytdl-core-discord is made by the __founder__ of Discord.js, let's wait and see first, it was most likely just made private on github and shows not found
